lib/computed_model.rb

```diff
@@ -307,24 +307,37 @@ def computing_plan(deps)
     end
   end
 
-  # @param deps [Array<Symbol, Hash>]
-  # @return [Hash{Symbol=>Array}]
-  def self.normalize_dependencies(deps)
-    normalized = {}
-    deps.each do |elem|
-      case elem
-      when Symbol
-        normalized[elem] ||= []
-      when Hash
-        elem.each do |k, v|
-          v = [v] if v.is_a?(Hash)
-          normalized[k] ||= []
-          normalized[k].push(*Array(v))
+  class << self
+    # @param deps [Array<Symbol, Hash>, Hash]
+    # @return [Hash{Symbol=>Array}]
+    def normalize_dependencies(deps)
+      normalized = {}
+      case deps
+      when Array
```

Comment:
How about doing `deps = Array(deps)`? That would simplify things and supports `deps: Symbol` at the same time.

Reply:
!!!!!

Reply:
fixed 611462f ✅